QUEST_LV_0100.tsv

```diff
@@ -6188,8 +6188,8 @@ QUEST_LV_0100_20150317_006187	Goddess Saule Follower Nojus is anxiously waiting
 QUEST_LV_0100_20150317_006188	$Find the 'ominous voice'
 QUEST_LV_0100_20150317_006189	$Nojus told you where the ominous voice is coming from. Go and find it.
 QUEST_LV_0100_20150317_006190	{memo X}
-QUEST_LV_0100_20150317_006191	$Defeated the Poatas summoned by the watcher. Go deep into the Thorn Forest to find Bramble.
-QUEST_LV_0100_20150317_006192	Defeat the summoned Poatas
+QUEST_LV_0100_20150317_006191	$Defeated the Poata summoned by the watcher. Go deep into the Thorn Forest to find Bramble.
```

Comment:
I have changed this line in my PR as well, however, mine is:

"Defeated the Poata summoned by the Beholder. Go deep into the Thorny Forest to find Bramble."